os/android/drmhwctwo.h

```diff
@@ -171,7 +171,7 @@ class DrmHwcTwo : public hwc2_device_t {
 
    private:
     //True after validateDisplay
-    bool checkValidateDisplay;
+    bool checkValidateDisplay = false;
```

Comment:
okay. Did you mean that the field bool checkValidateDisplay = false should be part of private variables of public hwc2_device_t ? I was thinking since, I'm going to use checkValidateDisplay in only the functions of class HwcDisplay; It is enough to keep the variable in class HWCDisplay.
As I'm also new to C++ programming, I may not have interpreted the code correctly. 
Thanks for the review comments.